src/orange/LevelStateAccessor.hh

```diff
@@ -0,0 +1,125 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021-2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/LevelStateAccessor.hh
```

Comment:
I think this class should probably be a detail...

Reply:
I thought the same thing about the Translator class, but you wanted that in the top-level dir. Should I move both to detail?

Reply:
Well, I think Translator (and its future brother Transformer?) could be reusable elsewhere, rather than just an implementation detail of the tracking. The LSA is purely an implementation detail.